src/api/response/iiif/collection.js

```diff
@@ -112,12 +133,12 @@ function loadItem(item) {
     },
     thumbnail: [
       {
-        id: [`${item.representative_file_set.url}/full/400,/0/default.jpg`],
+        id: `${item.representative_file_set.url}/full/400,/0/default.jpg`,
         service: [
           {
             profile: "http://iiif.io/api/image/2/level2.json",
             "@context": "http://iiif.io/api/image/2/context.json",
-            "@id": `${item.representative_file_set.url}`,
+            "@id": item.representative_file_set.url,
           },
```

Comment:
1.  Remove the service? 
2. Use `item.thumbnail` as `id`?
3. Add format as `image/jpeg`